source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -438,60 +457,6 @@ def get_stop_area(self, obj):
     def get_lines(self, obj):
         return LineSerializer(obj.lines, many=True, display_none=False).data
 
-    def get_access_points(self, obj):
-        return AccessPointSerializer(obj.access_points, many=True, display_none=False).data
-
-
-class AccessPointSerializer(PbGenericSerializer):
-    coord = CoordSerializer(required=False)
-    is_entrance = jsonschema.MethodField(schema_type=bool, display_none=False)
-    is_exit = jsonschema.MethodField(schema_type=bool, display_none=False)
-    pathway_mode = jsonschema.MethodField(schema_type=int, display_none=False)
-    length = jsonschema.MethodField(schema_type=int, display_none=False)
-    traversal_time = jsonschema.MethodField(schema_type=int, display_none=False)
-    stair_count = jsonschema.MethodField(schema_type=int, display_none=False)
-    max_slope = jsonschema.MethodField(schema_type=int, display_none=False)
-    min_width = jsonschema.MethodField(schema_type=int, display_none=False)
-    signposted_as = jsonschema.MethodField(schema_type=str, display_none=False)
-    reversed_signposted_as = jsonschema.MethodField(schema_type=str, display_none=False)
-    parent_station = jsonschema.MethodField(schema_type=lambda: StopAreaSerializer(), display_none=False)
-
-    def get_is_entrance(self, obj):
```

Comment:
if you erase `get` function inside the AccessPointSerializer, you will have troubles with that... I tested it and if the `get` no exists we print empty string and -1 value... So this is mandatory.